assets/css/main.css

```diff
@@ -170,6 +170,28 @@ body {
   text-decoration: none;
```

Comment:
Can this file have its extension restored to `.scss`?
This is a [Sass file](https://sass-lang.com/guide) which eventually compiles into CSS. If it is a `.css` file, the compilation doesn't happen to it and the styles break.